apps/readest-app/src/services/webAppService.ts

```diff
@@ -141,25 +141,6 @@ const indexedDBFileSystem: FileSystem = {
   async removeDir() {
     // Directories are virtual in IndexedDB; no-op
   },
-  async readDir(path: string) {
```

Comment:
The web version needs this API to delete books.

Reply:
I double checked this. Currently this API is not used by other functions. But in the future there will be places that utilize readDir in web app for example when book files need pre/post-processing and there are some intermediate caches/documents generated when reading. We need a way to list a directory in order to keep the API consistent across platforms. An optimized implementation is needed of course for the web file system. The indexedDB approach is not performant and [OPFS](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API/Origin_private_file_system) might be an option.